buildSrc/src/main/java/Config.kt

```diff
@@ -58,7 +60,8 @@ object Config {
     }
 
     object Sentry {
-        val SENTRY_CLIENT_NAME = "sentry.java.android"
+        val SENTRY_JAVA_CLIENT_NAME = "sentry.java"
+        val SENTRY_ANDROID_CLIENT_NAME = "$SENTRY_JAVA_CLIENT_NAME.android"
```

Comment:
Doesn't the string format need to be like this?
```suggestion
        val SENTRY_ANDROID_SDK_NAME = "${SENTRY_JAVA_CLIENT_NAME}.android"
```

Reply:
Done